docs/7-vector-search/5-aws-sagemaker.mdx

```diff
@@ -0,0 +1,109 @@
+# 🦸 Using Amazon SageMaker
+
+[Amazon SageMaker](https://aws.amazon.com/pm/sagemaker/) is a cloud based machine-learning platform that enables developers to build, train, and deploy machine learning (ML) models for any use case with fully managed infrastructure, tools, and workflows.
+It also enables developers to deploy ML models on embedded systems and edge-devices. SageMaker was launched in November 2017 and has been improved ever since, especially since the increased popularity and demand of ML and AI systems in recent years.
+
+To make it easier to get started, Amazon SageMaker JumpStart provides a set of solutions for the most common use cases that can be deployed readily with just a few clicks.
+
+Amazon SageMaker is available for free, for 2 months, as part of the AWS Free Tier program. Users can get access to 250 hours per month of `ml.t3.medium` notebooks usage with the Free Tier.
+
+<Screenshot src="img/screenshots/7-vector-search/5-aws-sagemaker/0-pricing.png" url="https://aws.amazon.com/pm/sagemaker/" alt="AWS SageMaker" />
+
+## Getting Started with Amazon SageMaker
+Amazon SageMaker JumpStart helps you quickly and easily get started with machine learning. The solutions are fully customizable and supports one-click deployment and fine-tuning of more than 150 popular open source models such as natural language processing, object detection, and image classification models.
+
+Popular solutions include:
+- Extract & Analyze Data: Automatically extract, process, and analyze documents for more accurate investigation and faster decision-making.
+- Fraud Detection: Automate detection of suspicious transactions faster and alert your customers to reduce potential financial loss.
+- Churn Prediction: Predict likelihood of customer churn and improve retention by honing in on likely abandoners and taking remedial actions such as promotional offers.
+- Personalized Recommendations: Deliver customized, unique experiences to customers to improve customer satisfaction and grow your business rapidly.
+
+## Let's set up a playground for you to try it out!
+
+On the main page of SageMaker you'll find the option to `Set up for single user` or `Set up for organization`. These will set up a domain and a quick start user.
+
+<Screenshot src="img/screenshots/7-vector-search/5-aws-sagemaker/1-main-page.png" url="https://aws.amazon.com/pm/sagemaker/" alt="AWS SageMaker" />
```

Comment:
Same note about screenshots. I try to keep only those who show an action that is hard to find on the UI, because every time I give a Lab, the UI changes the day before, and people are lost 😅